### PR TITLE
Fixed all grammar issues and typos in README.md

### PR DESCRIPTION
**Pull request by bishal7679**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   docs update 

- **Why was this change needed?** (You can also link to an open issue here)

   There are lot of grammar mistake and typos which should be fixed!

### CODE REVIEW

**Comment by bishal7679** [comment]
@jainpawan21 @davidsoderberg PTAL on this PR!

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The ultimate service for managing multi-channel notifications with a single API.
 
 ## ⭐️ Why Novu?
 
-Building a notification system is hard, at first, it seems like just sending an email, but in reality, it's just the beginning. Users today expect a multi-channel communication experience via email, SMS, push, chat, and other channels. An ever-growing list of providers is popping up daily, and notifications are spread around the code. Novu's goal is to simplify notifications and provide developers with the tools to create meaningful communication between the system and its users.
+Building a notification system is hard, at first, it seems like just sending an email, but in reality, it's just the beginning. Users today expect a multi-channel communication experience via email, SMS, push, chat, and other channels. An ever-growing list of providers is popping up daily, and notifications are spread around the world. Novu's goal is to simplify notifications and provide developers with the tools to create meaningful communication between a system and its users.
```
Comment (p-fernandez) [inline README.md:48]: Same comment as this previous PR with the same changes proposed: https://github.com/novuhq/novu/pull/1711#discussion_r995022955

File: README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ await novu.trigger('<TRIGGER_NAME>',
 
 ## Embeddable Notification Center
 
-Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
+Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an IFrame embed if you are not using React.
```
Comment (p-fernandez) [inline README.md:116]: Same comment as this previos PR with the same change proposed: https://github.com/novuhq/novu/pull/1711#discussion_r995021703

File: README.md [hunk 3]
```diff
@@ -125,7 +125,7 @@ Using the Novu API and admin panel you can easily add real-time notification cen
 
 ## Providers
 
-Novu provides a single API to manage providers across multiple channels with a simple to use interface.
+Novu provides a single API to manage providers across multiple channels with a convenient interface.
```
Comment (p-fernandez) [inline README.md:128]: Please rollback to the current copy. Thank you.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Comment by bishal7679** [comment]
@p-fernandez PTAL again!

**Review by scopsy — APPROVED** [review]

**Pull request merged**